VersionControlUI/src/nodes/DiffFrame.h

```diff
@@ -66,21 +66,30 @@ class VERSIONCONTROLUI_API DiffComparisonPair : public Super<Visualization::UINo
 		Model::Text* singleObjectPath();
 
 		bool twoObjectPathsDefined();
-		bool isDummyDiffComparisonPair();
+		bool isDummyDiffFrame();
 		QString comparisonName();
 
 		QList<ObjectPathCrumbData> objectPathCrumbsDataOldNode();
 		QList<ObjectPathCrumbData> objectPathCrumbsDataNewNode();
 
 		static QString computeObjectPath(Model::Node* node);
 
+		void addOldChangedNode(Model::Node* oldNode);
```

Comment:
I realize this was most likely accidental. Just be more careful.

To make this easier, it helps to make/edit local commits more often, and then just submit them, once you're happy with the local state.